src/libs/Environment/index.android.js

```diff
@@ -0,0 +1,57 @@
+import lodashGet from 'lodash/get';
+import Config from 'react-native-config';
+import CONST from '../../CONST';
+import {version} from '../../../package.json';
+
+let environment = CONST.ENVIRONMENT.PRODUCTION;
+let hasSetEnvironment = false;
```

Comment:
I think you could get rid of this variable altogether and just set `environement = null` to begin with. Would that work?

Reply:
Good call, that works!